### PR TITLE
Release hikes

### PR DESCRIPTION
**Pull request by BerkeleyTrue**
Updates jobs/hikes to new Thundercats-React
Fixes to gulp
Initial challenge completion logic

### CODE REVIEW

File: common/app/flux/Actions.js [hunk 1]
```diff
@@ -4,46 +4,48 @@ import debugFactory from 'debug';
...
-  },
+  getUser({ isPrimed }) {
+    if (isPrimed) {
+      return null;
+    }
 
-  getUser: null,
-  updateRoute(route) {
-    return { route };
+    debug('fetching user data');
```
Comment (ltegman) [inline common/app/flux/Actions.js]: I'm guessing this shouldn't still be here.

Comment (BerkeleyTrue) [inline common/app/flux/Actions.js]: Why not?

**Comment by BerkeleyTrue** [comment]
Please test:
beta.freecodecamp.com/jobs
and beta.freecodeamp.com/hikes

**Comment by ltegman** [comment]
I find the transition animation between questions to be a little bit too much, but I'm not running into any issues. I'll wait for another set of eyes, but from what I can see this is ready for merge.

**Comment by SaintPeter** [comment]
Hikes:
Mousing over the video gives me an `[object Object]` popup.
image
As @ltegman Mentioned, the transitions on the questions are comically over the top.  Just a simple slide would be fine.
There is no "up" navigation to take you back to the main hikes page.

Jobs:
Um .. . there are no jobs listed.
The checkout screen is pretty sketchy:
image
Needs to look more like an invoice.  What is `+1000`?

**Comment by BerkeleyTrue** [comment]
I've added some jobs to the DB. 

> The checkout screen is pretty sketchy:

**Comment by BerkeleyTrue** [comment]
The hover issue should be fixed now

**Comment by SaintPeter** [comment]
Ok, looks good, or at least no worse than it had been before.

**Comment by ltegman** [comment]
:shipit:

File: client/index.js [hunk 2]
```diff
@@ -6,9 +6,10 @@ import debugFactory from 'debug';
 import { Router } from 'react-router';
 import { createLocation, createHistory } from 'history';
 import { hydrate } from 'thundercats';
-import { Render } from 'thundercats-react';
+import { render$ } from 'thundercats-react';
```
Comment (Mimi2k15) [inline client/index.js]: thank you!!

**Pull request merged**